infrastructure/templates/infrastructure/powerplant_detail.html

```diff
@@ -121,9 +121,20 @@ <h3>Estimated Output</h3>
       <div class="row double">
           <section>
               <h3>Fuel</h3>
```

Comment:
If we stick with this format, we should suppress empty headers
<img width="361" alt="screen shot 2019-01-03 at 5 08 24 pm" src="https://user-images.githubusercontent.com/129679/50664183-3324fe80-0f7a-11e9-83d6-863dea5e99be.png">

Reply:
So based on Maesea response there is now only one of these per page.

So on Projects there is `fuel_type` and on Power Plants there is `fuel_category`